### ISSUE TITLE
Version control

### ISSUE THREAD

**Issue by brianlangseth-NOAA** [brianlangseth-NOAA/Spatial-Workshop-SPASAM#48]
@JDeroba , thanks for adding your additions to the most recent post. I noticed however we are working from different versions. I had merged Data_loading_JJD with Data_loading on Nov 8 in commit 4931b71. It appears your newest commit (e9db539) was based on the Data_loading_JJD formating so unintentionally reverts the changes I had made on Nov 8. 

Gien you have added a number of new changes, and your version should be one we go with, I will try to remake the changes I did on Nov 8 onto your newest version. These were almost solely organizational, e.g. keeping changes in the phase of selectivity in a single place in the code, with one exception for maintaining usability with the 4 fleet model option.

Main point: hold off on making further changes until I update the newest Data_loading file. If you have already made some updates, let me know. In fact, let me know if you haven't made any changes. That way we wont have to do this twice. Im hoping this wont affect you much at all and we can get back to one version without issue.

**Comment by brianlangseth-NOAA** [comment]
Ive made the updates. Right now Im running the model to confirm results are the same as YFT_1area_7fleets_12. If they are, and you haven't made any changes to Data_loading since after pushing your version, I will push.

**Comment by brianlangseth-NOAA** [comment]
I could not reproduce what you did in YFT_1area_7fleets_12 by running Data_loading.R. Is this a run with dataset 12 or the 12th iteration of your explorations? I ask because there were some noticeable differences in things like recruits_BM, biomass_population, and the catch at age proportions. The later is unexpected.

I also noticed differences in rec_type (was 1 now is 2), N_start (was 4 now 8), wt_F_pen (was 0 now 1), mean_N (was 11 now 8). I dont see these changes in the Data_loading.R file. 

Id like to have changes reflected in our Data_loading script so that we can reproduce things. It may just be as easy as adding those things in, however the values for biomass_population and recruits_BM are really odd, but may be due to the change from average recruitment to the SR curve (i.e. rec_type = 1 to rec_type = 2).

If you recall making those changes during your explorations, then I think we should add them to the 1 area model, and then I can make those adjustments for the 4 area model as well. Just, let me update my changes to Data_loading.R first though before making updates.

**Comment by JDeroba** [comment]
I have not made more changes and don't plan to. You can fix the problem and
we'll be all set.

apologies,
JJD

**Comment by brianlangseth-NOAA** [comment]
@JDeroba Will do. After some more sleuthing about the differences between your model run and mine, I realized I am to blame. My panmictic OM is not up to date with our github version, and is why my values differ from yours. This may resolve all the issues. 
I pull from a OM version from outside my github repo whereas you pull from the github repo as shown in [these lines](https://github.com/brianlangseth-NOAA/Spatial-Workshop-SPASAM/blob/67811d20bef5e41ab62e5174e2d8a4cecc9fde88/Data%20loading.R#L19-L34) of data_loading.R. I had always thought that version was the same, but that is not the case. I will update my version, which based on github, shows that I did not account for changes in commit eac3fe0.
